tests/test_bad_example.py

```
@@ -0,0 +1,339 @@
+def test_hardcoded_port():
+    """Fetch the model list from the serving endpoint."""
+    resp = requests.get("http://localhost:8000/v1/models")
+    assert resp.status_code == 200
+
+
+# 2. Hardcoded temp path
+# Rule: pytest-guidelines > DO NOT hardcode temp paths
+@pytest.mark.pre_merge
+@pytest.mark.gpu_0
+@pytest.mark.unit
+def test_hardcoded_temp_path(tmp_path):
```

Ruff (0.15.5)

[warning] Unused function argument: `tmp_path`

(ARG001)